tool/tsh/resolve_default_addr_test.go

```diff
@@ -233,8 +233,6 @@ func TestResolveUndeliveredBodyDoesNotBlockForever(t *testing.T) {
 }
 
 func TestResolveDefaultAddrTimeoutBeforeAllRacersLaunched(t *testing.T) {
-	t.Parallel()
-
```

Comment:
Disabled parallelism in this test because it spawns 1000 local listeners.
This usually hits a file descriptor limit on my local machine when other tests are also spawning connections/listeners.

Reply:
Good call, I just upped my file descriptor limit